helix-core/src/main/java/org/apache/helix/cloud/event/HelixEventHandlingUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.helix.cloud.event;
+
+import org.apache.helix.BaseDataAccessor;
+
+
+public class HelixEventHandlingUtil {
+
+  public static boolean enableInstanceForCloudEvent(String clusterName, String instanceName, boolean isEnable, long timeStamp,
```

Comment:
1. Open for discussion. I think our intention is to hide away these APIs from users, maybe we can restrict the access to package-only?
2. Making a note on our "return false" or "throw exception" discussion when the action is failed
3. Should we use the timestamp when the instance is actually enabled/disabled instead of when we initiate this move?

Reply:
Updated.